src/OpenLoco/Map/TileManager.cpp

```diff
@@ -321,9 +280,85 @@ namespace OpenLoco::Map::TileManager
 
         if (quad > 0)
         {
-            height.landHeight += quad / 2;
+            return quad / 2;
+        }
+        return 0;
+    }
+
+    /**
```

Comment:
I see this is in the original code, but interesting to see javadoc style comments here

Reply:
Yeah think this was a copy paste from OpenRCT2 but i tend to not bother filling in unless the function is very extensively used and has a few confusing arguments.